snuba/datasets/configuration/generic_metrics/storages/distributions_bucket_storage.yaml

```diff
@@ -0,0 +1,53 @@
+storage:
```

Comment:
I'd like some kind of header metadata in these YAML files similar to what I did in my PR that lets us do a quick sanity check of what we're trying to parse:

https://github.com/getsentry/snuba/pull/3037/files#diff-384b271ce3ab4bf13b8aa9156f53945508d69e6b59fd612a2e873a0675ecdcafR1-R4

I copied mine from Kubernetes and I don't really care if we follow that format precisely, but I think at the very least we should have schema version and type of file encoded at the top